From 07716417e7579250d04ee1b8e1bd8950335563d3 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@nxp.com>
Date: Wed, 16 Aug 2017 14:59:44 +0800
Subject: [PATCH 2411/5242] MLK-16065-14 ARM64: dts:
 fsl-imx8qm-lpddr4-arm2.dts: add USB Type-C port
 support

commit  d2cf4352e1773f55cc20586980a6653bb68e928b from
https://source.codeaurora.org/external/imx/linux-imx.git

Due to pin conflict with other modules, we can only enable
host mode for Type-C port. If the user wants to use both
host and device mode for Type-C port, please use
fsl-imx8qm-lpddr4-arm2-usb3.dts instead.

Signed-off-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts  |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
index b9552e1..fe8d369 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
@@ -479,6 +479,11 @@
 	status = "okay";
 };
 
+&usbotg3 {
+	dr_mode = "host";
+	status = "okay";
+};
+
 &fec1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_fec1>;
-- 
1.7.9.5

